### ISSUE TITLE
mopinion-react-native-sdk contains invalid configuration

### ISSUE THREAD

**Issue by tijs** [mopinion/mopinion-sdk-react-native#36]
I get the following warning when running the RN packager:

```
warn Package mopinion-react-native-sdk contains invalid configuration: "dependency.assets" is not allowed. Please verify it's properly linked using "react-native config" command and contact the package maintainers about this.
```

**Comment by ramonjaspers** [comment]
Still got this, have you found a solution to this warning @tijs?

**Comment by ozyildirim** [comment]
same situation, is there any update?

**Comment by Anwardo** [comment]
Issue was fixed in 3.0.0
